dist/main.js

```diff
@@ -0,0 +1,147 @@
+const renderer = new Renderer()
+const localStorageManager = new LocalStorageManager()
+
+const getRoster = function () {
+  const $input = $("#team-name-input")
+  if ($input.val().length) {
+    $.get(`/teams/${$input.val()}`, function (data) {
+      $input.val(null)
+      if (data.teammates.length) {
+        renderer.renderPlayers(data)
+      } else {
+        renderer.renderEmptyState(data.teamName)
+      }
+    })
+  } else {
+    renderer.renderAlertedInput($input)
+  }
+}
+
+$("#players-container").on("click", "button.showStats", function () {
+  $player = $(this).closest(".player")
+  const firstName = $player
+    .attr("data-firstName")
+    .replace(/[^a-zA-Z-+.^:,]+/g, "")
+    .toLowerCase()
+  const lastName = $player
+    .attr("data-lastName")
+    .replace(/[^a-zA-Z-+.^:,]+/g, "")
+    .toLowerCase()
+  const player = `${lastName}+${firstName}`
+
+  $.get(`/playerStats/${player}`, function (stats) {
+    renderer.renderPlayerStats($player, stats)
+  })
+})
+
+$("#players-container").on("click", ".statistics-header>.go-back", function () {
+  $player = $(this).closest(".player")
```

Comment:
When declaring a new variable you should declare it with _const_ or _let_, otherwise, the variable will be defined as "_var_"